templates/traefik.yaml

```diff
@@ -26,9 +26,6 @@ spec:
       service:
         labels:
           servicemonitor.kubeaddons.mesosphere.io/path: "metrics"
-      metrics:
```

Comment:
Hum, can you explain more about this change?

Does that mean `/metrics` won't be served by Traefik?

Reply:
Yeah, why do you remove this change?

Reply:
This will disable traefik of exposing metric's

Reply:
The whole stanza
```yml
```
was duplicated, you can see the 2nd one here: https://github.com/mesosphere/kubeaddons-configs/pull/124/files#diff-76e7e224155bfbf77cdffd73534adae5L39 

(Yeah, Github diff view is not helping here, as you have to unfold the code view somewhere there is not a diff).

Reply:
oh! I see @dduportal, yeah GH didn't help here. 👍

Reply:
@hectorj2f no problem, thanks for the eagle eyes here!